### PR TITLE
DataContext must be set in Load event.

### PR DESCRIPTION
**Pull request by aosyatnik**
This is [quick fix](https://github.com/DynamoDS/Dynamo/pull/3879).
I found the reason of error. I will leave comments in code to make it clear.

Reviewers
@Benglin , please take a look.

### CODE REVIEW

File: src/DynamoCoreWpf/Controls/DynamoTextBox.cs [hunk 1]
```diff
@@ -387,13 +387,6 @@ private void LoadMainDynamoWindow(object sender, RoutedEventArgs e)
 
         public void SetDataContext(object dataContext, bool closeImmediately = false)
         {
-            if (mainDynamoWindow == null)
```
Comment (aosyatnik) [inline src/DynamoCoreWpf/Controls/DynamoTextBox.cs]: Quick fix, should be removed.

File: src/DynamoCoreWpf/Views/Search/LibraryContainerView.xaml.cs [hunk 2]
```diff
@@ -42,7 +42,6 @@ public SearchView(SearchViewModel searchViewModel, DynamoViewModel dynamoViewMod
             viewModel = searchViewModel;
             this.dynamoViewModel = dynamoViewModel;
 
-            DataContext = viewModel;
```
Comment (aosyatnik) [inline src/DynamoCoreWpf/Views/Search/LibraryContainerView.xaml.cs]: DataContext is set in Initialization. Tooltip is bound to TargetUpdated. That means as soon as DataContext is set tooltip tries to show something. But main dynamo window is not loaded yet. And it fails.

Comment (aosyatnik) [inline src/DynamoCoreWpf/Views/Search/LibraryContainerView.xaml.cs]: Another problem:
if data context is not set, we have BindingExpressions in LibraryContainerView.
System.Windows.Data Error: 40 : BindingExpression path error: .......

Can we leave these binding errors not fixed?

Comment (aosyatnik) [inline src/DynamoCoreWpf/Views/Search/LibraryContainerView.xaml.cs]: Or we can leave everything as it is now: with quick fix and without binding errors.

File: src/DynamoCoreWpf/Controls/DynamoTextBox.cs (hunk 1 above)
Comment (Benglin) [inline src/DynamoCoreWpf/Controls/DynamoTextBox.cs]: You can remove this,  for sure. But please update the statement below to `if (mainDynamoWindow == null || !mainDynamoWindow.IsActive) return;`

**Comment by Benglin** [comment]
Thanks @aosyatnik, LGTM!

**Pull request merged**